pkg/yurthub/server/server.go

```diff
@@ -127,3 +172,41 @@ func healthz(w http.ResponseWriter, _ *http.Request) {
 	w.WriteHeader(http.StatusOK)
 	fmt.Fprintf(w, "OK")
 }
+
+// create a certificate manager for the yurthub server and run the csr approver for both yurthub
+// and generate a TLS configuration
+func genUseCertMgrAndTLSConfig(certificateMgr interfaces.YurtCertificateManager, certDir string, stopCh <-chan struct{}) (*tls.Config, error) {
```

Comment:
It's more readable to prepare `tls.config` after  `client certificate` ready instead of preparing when start server.

and the client certificate preparation ready is here: https://github.com/openyurtio/openyurt/blob/master/cmd/yurthub/app/start.go#L95